src/interceptors/WebSocket/WebSocketServer.ts

```diff
@@ -0,0 +1,33 @@
+import type { WebSocketSendData } from './WebSocketTransport'
+import type { WebSocketMessageListener } from './implementations/WebSocketClass/WebSocketClassInterceptor'
+
+/**
+ * The WebSocket server instance represents the actual production
+ * WebSocket server connection. It's idle by default but you can
+ * establish it by calling `server.connect()`.
+ */
+export class WebSocketServer {
```

Comment:
Similar to the WebSocket client representation, this is the actual WebSocket server connection representation. It's used to:

- Establish the original connection to the server;
- Intercept the incoming server events (the real ones);
- Forward any (intercepted) outgoing client events to the actual server (if you wish a full passthrough interception).